packages/components/src/Chips/InternalChipDismissible/__tests__/KeyboardNavigation.test.tsx

```diff
@@ -0,0 +1,252 @@
+
+      // Move to middle chip with right arrow
+      await userEvent.keyboard("{ArrowRight}");
+      const middleChip = getByChipLabelText("middle");
+      expect(middleChip).toHaveFocus();
+
+      // Move to last chip with right arrow
+      await userEvent.keyboard("{ArrowRight}");
+      const lastChip = getByChipLabelText("last");
+      expect(lastChip).toHaveFocus();
+
+      // Move to Add button with right arrow
+      await userEvent.keyboard("{ArrowRight}");
+      const addButton = screen.getByRole("button", { name: "Add" });
+      expect(addButton).toHaveFocus();
+    });
+
+    it("should navigate from Add button to last chip with left arrow", async () => {
+      // Navigate to Add button first
+      await userEvent.tab();
+      await userEvent.keyboard("{ArrowRight}");
+      await userEvent.keyboard("{ArrowRight}");
+      await userEvent.keyboard("{ArrowRight}");
+
+      const addButton = screen.getByRole("button", { name: "Add" });
+      expect(addButton).toHaveFocus();
+
+      // Move back to last chip with left arrow
+      await userEvent.keyboard("{ArrowLeft}");
+      const lastChip = getByChipLabelText("last");
+      expect(lastChip).toHaveFocus();
+    });
+
+    it("should navigate from last chip to first chip with left arrow", async () => {
+      // Focus the last chip first
+      await userEvent.tab();
+      await userEvent.keyboard("{ArrowRight}");
+      await userEvent.keyboard("{ArrowRight}");
+
+      const lastChip = getByChipLabelText("last");
+      expect(lastChip).toHaveFocus();
+
+      // Move back to middle chip with left arrow
+      await userEvent.keyboard("{ArrowLeft}");
+      const middleChip = getByChipLabelText("middle");
+      expect(middleChip).toHaveFocus();
+
+      // Move back to first chip with left arrow
+      await userEvent.keyboard("{ArrowLeft}");
+      const firstChip = getByChipLabelText("first");
+      expect(firstChip).toHaveFocus();
+    });
+
+    it("should not navigate past first chip with left arrow", async () => {
+      // Focus the first chip
+      const firstChip = getByChipLabelText("first");
+      await userEvent.tab();
+      expect(firstChip).toHaveFocus();
+
+      // Try to go before first chip (should stay on first)
+      await userEvent.keyboard("{ArrowLeft}");
+      expect(firstChip).toHaveFocus();
+    });
+
+    it("should not navigate past Add button with right arrow", async () => {
+      // Navigate to Add button
+      await userEvent.tab();
+      await userEvent.keyboard("{ArrowRight}");
+      await userEvent.keyboard("{ArrowRight}");
+      await userEvent.keyboard("{ArrowRight}");
+
+      const addButton = screen.getByRole("button", { name: "Add" });
+      expect(addButton).toHaveFocus();
+
+      // Try to go past Add button (should stay on Add)
+      await userEvent.keyboard("{ArrowRight}");
+      expect(addButton).toHaveFocus();
+    });
+  });
+
+  describe("Focus Management When Deleting Chips", () => {
```

Comment:
this deletion describe already exists in the other file. merging them seems better than having the coverage duplicate/split.